javascript/statements.json

```
@@ -562,7 +562,14 @@
                 "version_added": false
               },
               "nodejs": {
-                "version_added": null
+                "version_added": "8.5.0",
+                "flags": [
+                  {
+                    "name": "--experimental-modules",
+                    "type": "runtime_flag"
+                  }
+                ],
+                "notes": "files must have suffix .mjs, not .js"
```

Suggesting some changes to this note, to help it stand alone better and to format the file name text. What do you think?

```suggestion
                "notes": "Module file names must end with <code>.mjs</code>, not </code>.js</code>."
```

---

That's good. I added these changes and also changed the notes for `import` as well (which is where I got the notes from).